data/morphology/grab_unimorph_data.py

```diff
@@ -1,52 +1,69 @@
 #!/usr/bin/env python
 """Downloads UniMorph morphological paradigms data."""
-
 import json
 import logging
 import os
-import requests
 import time
-
 from typing import Dict
 
 
+""" import requests in separate code block because of third party status """
+import requests
+
+
+start_time = time.time()
+
+
 UNIMORPH_DICT_PATH = "unimorph_languages.json"
 
 
-def download(data_to_grab: Dict[str, str]) -> Dict[str, str]:
-    to_retry = {}
-    os.mkdir("tsv")
-    for language, url in data_to_grab.items():
-        with requests.get(url, stream=True) as response:
-            logging.info("Downloading: %s", language)
+def download():
+    os.makedirs("unimorph_for_split", exist_ok=True)
+    with open(UNIMORPH_DICT_PATH) as jfile:
+        data_to_grab = json.load(jfile)
+    urls = list(data_to_grab.values())
```

Comment:
This list casting on lines 24-25 then using range-for is totally unnecessary. Please revert to what was done on line 19...just loop over the `items()` pairs.